Dockerfile

```diff
@@ -56,6 +56,9 @@ USER spacelift
 
 FROM base AS azure
 
+RUN apk add --no-cache py3-pip gcc musl-dev python3-dev libffi-dev openssl-dev cargo make
+RUN pip install --break-system-packages azure-cli
```

Comment:
can we uninstall what we don't need anymore? these libraries could take gigabytes of space

Reply:
great idea. 

> also remove apk pkg cache

Note that Marcin already uses `--no-cache` so that one is at least covered 😄